.github/workflows/cleanup-after-pr.yml

```diff
@@ -19,6 +19,47 @@ jobs:
       - name: Log in to Azure ACR
         run: az acr login -n bitwardenqa
 
+      ########## Remove Docker images ##########
+      - name: Remove the Docker image from bitwardenqa ACR
+        env:
+          REF: ${{ github.event.pull_request.head.ref }}
+          REGISTRY: bitwardenqa
+          SERVICES: |
+            services:
+              - Admin
+              - Api
+              - Attachments
+              - Events
+              - EventsProcessor
+              - Icons
+              - Identity
+              - K8S-Proxy
+              - MsSql
+              - Nginx
+              - Notifications
+              - Server
+              - Setup
+              - Sso
+        run: |
+          for SERVICE in $(echo "${{ env.SERVICES }}" | yq e ".services[]" - )
+          do
+            SERVICE_NAME=$(echo $SERVICE | awk '{print tolower($0)}')
+            IMAGE_TAG=$(echo "${REF}" | sed "s#/#-#g")  # slash safe branch name
+
+            echo "[*] Checking if remote exists: $REGISTRY.azurecr.io/$SERVICE_NAME:$IMAGE_TAG"
+            TAG_EXISTS=$(
+            az acr repository show-tags --name $REGISTRY --repository $SERVICE_NAME \
+            | jq --arg $TAG "$IMAGE_TAG" -e '. | any(. == "$TAG")'
+            )
+
+            if [[ "$TAG_EXISTS" == "true" ]]; then
+              echo "[*] Tag exists. Removing tag"
+              az acr repository delete --name $REGISTRY --image $SERVICE_NAME:$IMAGE_TAG --yes
+            else
+              echo "[*] Tag does not exist. No action needed"
+            fi
+          done
+
```

Comment:
This registry isn't even used, only `bitwardenprod` https://github.com/bitwarden/server/blob/5a5e5c50580055f3dd681cf763627b0324d73e82/.github/workflows/build.yml#L13-L14. I think this, along with other references, can be removed from this workflow. Feel free to handle what is outside of scope in a different change, but this specific one can be removed.

Reply:
Thanks for the suggestion!